NetSDK/Services/Parsing/Classes/SplitParser.cs

```diff
@@ -31,7 +31,7 @@ public ParsedSplit Parse(Split split)
                     conditions = new List<ConditionWithLogic>(),
                     changeNumber = split.changeNumber,
                     trafficTypeName = split.trafficTypeName,
-                    algo = split.algo
+                    algo = split.algo == 0 ? AlgorithmEnum.LegacyHash : (AlgorithmEnum)split.algo
```

Comment:
@patricioe When algo is not specified (null), as split.algo is an int it will take its default value (0).
If it is possible that we will have something like 
`'algo': null`
 in our json, yes I will need to handle that